packages/marko-web/express/error-handlers.js

```diff
@@ -47,15 +49,26 @@ const redirectOrError = ({
     } else {
       render(res, { statusCode, err, template });
     }
-  }).catch(() => render(res, { statusCode, err, template }));
+  }).catch(() => {
+    try {
+      render(res, { statusCode, err, template });
+    } catch (e) {
+      fatalErrorHandler(err);
```

Comment:
This needs to be `e` not `err`

Reply:
Actually in this case it doesn't -- `e` is the final fatal error (headers already sent) that caused the error template not to render. `err` is the original error that would have been printed in the error template.